test/lint/lint-git-commit-check.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Linter to check that commit messages have a new line before the body
+# or no body at all
+
+import argparse
+import os
+import sys
+
+from subprocess import check_output
+
+
+def parse_args():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(
+        description="""
+            Linter to check that commit messages have a new line before
+            the body or no body at all.
+        """,
+        epilog="""
+            You can manually set the commit-range with the COMMIT_RANGE
+            environment variable. Defaults to current merge base when
+            neither prev-commits nor the environment variable is set.
```

Comment:
I think it will be helpful to the user if we add an usage example for `COMMIT_RANGE` environment variable here, just how the help manual for `test/lint/lint-git-commit-check.sh` does it,  something like: 
`COMMIT_RANGE='b69fd5e...2e6cfc7' test/lint/lint-git-commit-check.py`

Reply:
Yes, you're right!

Reply:
نعم  انت علي حق